football-gui-game/src/main/java/com/github/lipinskipawel/controller/OneVsLanController.java

```diff
@@ -37,13 +41,25 @@ public void rightClick(final RenderablePoint renderablePoint) {
      * invokes the {@link Table#drawBoard(BoardInterface, Player)}.
      */
     void startServer() {
-        server = Server.Companion.createServer(SERVER_PORT)
-                .onReceived(move -> {
-                    move.getMove().forEach(System.out::println);
-                    this.gameFlowController = this.gameFlowController.makeAMove(move);
-                    this.table.drawBoard(this.gameFlowController.board(), this.gameFlowController.player());
-                    return null;
+        final var executor = Executors.newSingleThreadExecutor();
```

Comment:
This code is only for resolving the compilation errors.
Currently there is no option in the App of playing via LAN.